op-deployer/pkg/deployer/bootstrap/consolidated.go

```diff
@@ -0,0 +1,263 @@
+package bootstrap
+
+import (
+	"context"
+	"crypto/ecdsa"
+	"fmt"
+	"strings"
+
+	"github.com/ethereum-optimism/optimism/op-deployer/pkg/deployer"
+	"github.com/ethereum-optimism/optimism/op-deployer/pkg/deployer/artifacts"
+	"github.com/ethereum-optimism/optimism/op-deployer/pkg/deployer/opcm"
+	"github.com/ethereum-optimism/optimism/op-deployer/pkg/deployer/standard"
+	"github.com/ethereum-optimism/optimism/op-service/ctxinterrupt"
+	"github.com/ethereum-optimism/optimism/op-service/ioutil"
+	"github.com/ethereum-optimism/optimism/op-service/jsonutil"
+	oplog "github.com/ethereum-optimism/optimism/op-service/log"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/ethereum/go-ethereum/log"
+	"github.com/ethereum/go-ethereum/params"
+	"github.com/urfave/cli/v2"
+)
+
+// ConsolidatedOutput combines the outputs from Superchain and Implementations bootstrap
+type ConsolidatedOutput struct {
+	Superchain      opcm.DeploySuperchainOutput      `json:"superchain"`
+	Implementations opcm.DeployImplementationsOutput `json:"implementations"`
+}
+
+type ConsolidatedConfig struct {
+	L1RPCUrl                        string
+	PrivateKey                      string
+	Logger                          log.Logger
+	ArtifactsLocator                *artifacts.Locator
+	Outfile                         string
+	MIPSVersion                     int
+	WithdrawalDelaySeconds          uint64
+	MinProposalSizeBytes            uint64
+	ChallengePeriodSeconds          uint64
+	ProofMaturityDelaySeconds       uint64
+	DisputeGameFinalityDelaySeconds uint64
+	SuperchainProxyAdminOwner       common.Address
+	ProtocolVersionsOwner           common.Address
+	Guardian                        common.Address
+	Paused                          bool
+	RequiredProtocolVersion         string
+	RecommendedProtocolVersion      string
+	UseInterop                      bool
+	CacheDir                        string
+
+	privateKeyECDSA *ecdsa.PrivateKey
+}
+
+func (c *ConsolidatedConfig) Parse() error {
```

Comment:
Can you rename this method to `Check()` to align with similar functions on other config objects?